ext/rugged/rugged.h

```diff
@@ -147,6 +147,12 @@ struct rugged_cb_payload
     int exception;
 };
 
+struct rugged_treecount_cb_payload
+{
+	unsigned int count;
+	unsigned int limit;
```

Comment:
Totally correct, was just chatting about that with @brianmario. I think just `int` is probably fine. What are the portability concerns with `ssize_t`?